core/contracts/financial_templates/PricelessPositionManager.sol

```diff
@@ -25,12 +27,16 @@ contract PricelessPositionManager is Testable {
      */
     struct PositionData {
         bool isValid;
-        FixedPoint.Unsigned collateral;
         FixedPoint.Unsigned tokensOutstanding;
         // Tracks pending withdrawal requests. A withdrawal request is pending if `requestPassTimestamp != 0`.
         uint requestPassTimestamp;
         FixedPoint.Unsigned withdrawalRequestAmount;
+        // Raw collateral value. This value should never be accessed directly -- always use _getCollateral().
+        // To add or remove collateral, use _addCollateral() and _removeCollateral().
+        FixedPoint.Unsigned rawCollateral;
```

Comment:
Just so I understand, `rawCollateral` represents the pre-fee collateral.

If the contract has been running for a while and has accumulated 50% fees, then if I were to open a new position with $10, then my pre-fee collateral would be represented as $20?

The invariant is:
`(feeAdjustment) * Sum(rawCollateral) == totalPositionCollateral`

Reply:
That's correct.